### PR TITLE
Capstone reflections document

### PR DESCRIPTION
**Pull request by Hamled**
This is currently a first draft, I'm looking for reviews focused on:

1. What additional questions/prompts should we give them
1. Is more context required
1. Should there be some kind of post-script thing following the list of prompts

Things we probably need:

1. Instructions on how to fill out the prompts and I guess submit them to us?

### CODE REVIEW

File: reflections.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Capstone Reflections
+With any large project that you've undertaken, especially one which requires a lot of personal growth and self-improvement, it's important to take time to reflect on the your process for completing the project and the outcomes of the decisions you made along the way.
+
+We're providing a set of suggested reflection prompts / questions that you can use to consider how things went with your capstone project, and we encourage you to add your own prompts as well!
+
+## Reflection prompts
+1. What went well during the capstone project?
```
Comment (kariabancroft) [inline reflections.md]: I think the context of this that we discussed was before the end of capstones and potentially during the third week

File: reflections.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+1. What didn't go as well?
+1. What aspects of the project were more difficult than anticipated?
+1. What aspects were less difficult than anticipated?
+1. What features in your project were not possible in the time frame provided? Were any of them critical to the success of the project?
+
+1. How did your project plan shift as the project progressed?
+1. If any project features took longer than anticipated to complete, how did you modify the rest of your planned features to accommodate having less time?
+1. How did your intended plan of action line up with the reality?
+
+1. Is there anything you would do differently for the next large project you undertake?
```
Comment (kariabancroft) [inline reflections.md]: I think these questions are good - I think we are missing some reflections on learning goals which I think is really the most important. I'd like the overall impact of the reflection questions to be less about the capstone project itself and more about what the takeaways are going to be towards internship and future projects.

File: reflections.md [hunk 3]
```diff
@@ -4,14 +4,16 @@ With any large project that you've undertaken, especially one which requires a l
 We're providing a set of suggested reflection prompts / questions that you can use to consider how things went with your capstone project, and we encourage you to add your own prompts as well!
 
 ## Reflection prompts
-1. What went well during the capstone project?
-1. What didn't go as well?
+1. What did you learn about:
```
Comment (kariabancroft) [inline reflections.md]: These are great!!

File: reflections.md [hunk 4]
```diff
@@ -4,14 +4,16 @@ With any large project that you've undertaken, especially one which requires a l
...
+  * Debugging unfamiliar technologies and complex interactions between components?
+
 1. What aspects of the project were more difficult than anticipated?
 1. What aspects were less difficult than anticipated?
-1. What features in your project were not possible in the time frame provided? Were any of them critical to the success of the project?
-
-1. How did your project plan shift as the project progressed?
-1. If any project features took longer than anticipated to complete, how did you modify the rest of your planned features to accommodate having less time?
-1. How did your intended plan of action line up with the reality?
+1. What features in your project were not possible in the time frame provided? Why?
```
Comment (kariabancroft) [inline reflections.md]: I think this question is good - but I'm not sure what you'd expect out of it. It seems like we'd be most likely to get "not enough time".

Comment (Hamled) [reply reflections.md]: I think you're right. It probably needs some rephrasing if it's going to be meaningful.

I was hoping to get students to consider which features took longer than they had planned for and thus left them without enough time to complete other (hopefully less critical) things.

**Pull request merged**